### PR TITLE
Swkbd Applet Fixes

### LINKED ISSUE #5233: [Bug] Alphabet keyboard applet not working as intended
### Description of the issue

alphabet mode on the keyboard applet doesn't accept number or spaces, this is most noticeable on pokemon

ryujinx:
image
image

Switch:
2023060504070000-50E2A11CE4BDDC72EF99DF78315D4938

### Reproduction steps

try to nickname a pokemon

pokemon sword save: 
[000000000000004a.zip](https://github.com/Ryujinx/Ryujinx/files/11653012/000000000000004a.zip)

### Log file

N/A

### OS

Arch Liinux

### Ryujinx version

1.1.866

### Game version

1.3.1

### CPU

AMD Ryzen 7900X3D

### GPU

AMD RX 6600 XT

### RAM

64GB

### List of applied mods

_No response_

### Additional context?

_No response_

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
Should fix #5233

### CODE REVIEW

File: src/Ryujinx.Ava/UI/Applet/SwkbdAppletDialog.axaml.cs [hunk 1]
```diff
@@ -146,7 +146,7 @@ private void SetInputValidation(KeyboardMode mode)
                 case KeyboardMode.Alphabet:
                     localeText = LocaleManager.Instance.UpdateAndGetDynamicValue(LocaleKeys.SoftwareKeyboardModeAlphabet);
                     validationInfoText = string.IsNullOrEmpty(validationInfoText) ? localeText : string.Join("\n", validationInfoText, localeText);
-                    _checkInput = text => text.All(char.IsAsciiLetter);
+                    _checkInput = text => text.All(value => !CJKCharacterValidation.IsCJK(value) );
```
Comment (AcK77) [inline src/Ryujinx.Ava/UI/Applet/SwkbdAppletDialog.axaml.cs:149]:
```suggestion
                    _checkInput = text => text.All(value => !CJKCharacterValidation.IsCJK(value));
```

File: src/Ryujinx/Ui/Applet/SwkbdAppletDialog.cs [hunk 2]
```diff
@@ -93,8 +93,8 @@ public void SetInputValidation(KeyboardMode mode)
                     _checkInput = text => text.All(char.IsDigit);
                     break;
                 case KeyboardMode.Alphabet:
-                    _validationInfoText += "<i>Must be alphabets only.</i>";
-                    _checkInput = text => text.All(char.IsAsciiLetter);
+                    _validationInfoText += "<i>Must be non CJK-characters only.</i>";
+                    _checkInput = text => text.All(value => !CJKCharacterValidation.IsCJK(value) );
```
Comment (AcK77) [inline src/Ryujinx/Ui/Applet/SwkbdAppletDialog.cs:97]:
```suggestion
                    _checkInput = text => text.All(value => !CJKCharacterValidation.IsCJK(value));
```

**Review by AcK77 — COMMENTED** [review]
Idk if it fixes the issue, but just some little nits

**Comment by piplup55** [comment]
does indeed fix the issue outlined in #5233 
image
image

**Comment by WilliamWsyHK** [comment]
It does have different meaning than specified in the enum name. Interesting.

It also means that the below locales are also needed to be updated.
src/Ryujinx.Ava/Assets/Locales/zh_TW.json: `"SoftwareKeyboardModeAlphabet": "只接受非中日韓文字"`
src/Ryujinx.Ava/Assets/Locales/zh_CN.json: `"SoftwareKeyboardModeAlphabet": "只接受非中日韩文字"`

**Comment by WilliamWsyHK** [comment]
Seems like need to wait for another PR to happen then
Ok I will also update there if available as well

**Review by riperiperi — APPROVED** [review]
Functionally looks good, though I'm not sure about asking the user for non-CJK characters as they might not know what that means.

**Pull request merged**